README.md

```diff
@@ -2,62 +2,123 @@
 
 # End to end tests for Archive.org using [Playwright](https://playwright.dev/)
 
+
 ## (Optional) BrowserStack Local Setup
 
-- login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
+- Login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
 
 
-## Running tests locally
+## Local Setup
 
-- install dependencies:
+- Install dependencies:
 
     `npm i`
 
-- install Playwright browser libs:
+- Install Playwright browser libs:
 
     `npx playwright install`
 
-- run all the tests in headless mode and generate 1 whole test report:
+- Create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing
+
+    - this is required to run tests with loggedIn flows
+
+
+## Running tests
+
+- Run all tests: `npm run test`
+
+- Run tests using `generate` command:
+
+    - See `scripts/generateCommand.js`, update the arguments that starts at `line:36`
+
+    - Run the command: `npm run generate:command`, run the generated by the command
+
+- Run tests using custom execute script:
+
+    > Note:
+    >   - This will run a custom script that can take different shell script arguments/parameters for the following:
+    >   - The arrangement/order for passing of arguments can be jumbled for flexibility
+    >   - It's important to add `--` convention in CLI after the `npm command`
+    >       - It will separate npm-specific arguments from the arguments that are intended for the nodejs script
```

Comment:
QUESTION: So arguments before the `--` are for... npm? And the arguments are the `--` are for the "nodejs script"? Let's make this line clearer by saying specifically what the arguments before the `--` are for and what the ones after the `--` are for.

e.g. What is the "nodejs script"? Is that the JS configured in the package.json file? And for the arguments before `--`, are those for the `npm` command? Or the `npm run` command? Or `npm run execute`? You see what I mean? Let's be clearer about exactly which command the arguments are for.